Charith/example-5/data-gathering.html

```diff
@@ -11,51 +12,112 @@
-<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
-        integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
-        crossorigin="anonymous"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
-        integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
-        crossorigin="anonymous"></script>
-<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
-        integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
-        crossorigin="anonymous"></script>
+
+    <!-- Optional JavaScript -->
+    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+            integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+            crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+            integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+            crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+            integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+            crossorigin="anonymous"></script>
+
+    <!--    java script area begin-->
+    <script type = "text/javascript">
+        function validate() {
+
+           // alert( document.myForm.first_name.value);
+           // return false;
+            if( document.myForm.first_name.value === "" ) {
+                alert( "Please provide your first name!" );
+                document.myForm.Name.focus() ;
+                return false;
+            }
+            if( document.myForm.LAST_NAME.value === "" ) {
+                alert( "Please provide your Last name!" );
+                document.myForm.LAST_NAME.focus() ;
+                return false;
+            }
+            if( document.myForm.EMAIL.value === "" ) {
+                alert( "Please provide your Email!" );
+                document.myForm.EMAIL.focus() ;
+                return false;
+            }
+            if( document.myForm.PHONE_NUMBER.value === "" ) {
+                alert( "Please provide your Phone number!" );
+                document.myForm.PHONE_NUMBER.focus() ;
+                return false;
+            }
+            if( document.myForm.CITY.value === "" ) {
+                alert( "Please provide your City!" );
+                document.myForm.CITY.focus() ;
+                return false;
+            }
+            if( document.myForm.MESSAGE.value === "" ) {
+                alert( "Please provide your City!" );
+                document.myForm.MESSAGE.focus() ;
+                return false;
+            }
+            return( true );
+            function validateEmail() {
```

Comment:
We don't need to validate email manually if we use input type as email, either way, this is good try